frontend/src/components/DashboardPage.js

```diff
@@ -89,6 +92,15 @@ const DashboardPage = ({
               )}
             </Row>
           </Card>
+          <div style={{ display: 'flex', marginBottom: '1em' }}></div>
```

Comment:
The card placement on the Dashboard is a bit weird. Maybe place it in the second column below the samples ?